### PR TITLE
suggestions for page index (mostly)

### PR DESCRIPTION
**Pull request by christinerose**
Testing how the first sentence looks bolded to make it stand out more. Also, the page index list looked strange to me, as it mostly just restated what the links said, so I turned them into sentences. Looking forward to your thoughts @panglesd!

### CODE REVIEW

File: doc/odoc.mld [hunk 1]
```diff
@@ -45,16 +45,16 @@ For guidance on how to document your OCaml project, see {{!page-odoc_for_authors
...
+- {!page-odoc_for_authors} gives information for [odoc] authors.
+- {!page-cheatsheet} provides a quick quide for [odoc] markdown.
+- {!page-features} explores [odoc]'s language features.
+- {!page-driver} describes how [odoc] produces documentation.
+- {!page-ocamldoc_differences} outlines differences from OCamldoc.
+- {!page-dune} shows how to create docs using Dune.
+- {!page-parent_child_spec} delineates parent/child specifications.
+- {!page-interface} describes [odoc]'s public-facing interface and their support guarantees.
+- {!page-ocamlary} demonstrates the rendering of most of the OCaml constructs.
+- {!page-api_reference} lists [odoc]'s API reference library.
```
Comment (Julow) [inline doc/odoc.mld:60]: `api_reference` doesn't resolve but that's not related to this PR.

**Review by Julow — COMMENTED** [review]
Awesome! I love "delineate".

File: doc/odoc.mld (hunk 1 above)
Comment (christinerose) [reply doc/odoc.mld:60]: So does my suggestion work here, or does it need to be altered?

Comment (Julow) [reply doc/odoc.mld:60]: It works!

**Review by Julow — APPROVED** [review]
Let's merge!

**Pull request merged**